src/App.css

```diff
@@ -106,6 +106,27 @@
   align-self: flex-end;
 }
 
+/* Footer Component */
+footer {
```

Comment:
Please use a class name instead of the element name

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
fixed ✔

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good